tools/mksurfdata_esmf/gen_mksurfdata_namelist.xml

```diff
@@ -25,7 +25,7 @@
 
   <mksrf_fsoicol>
     <entry>
-      <data_filename>lnd/clm2/rawdata/pftcftdynharv.0.25x0.25.LUH2.histsimyr1850-2015.c20230226/mksrf_soilcolor_ctsm52_histLUH2_2005.c20230226.nc</data_filename>
+      <data_filename>/glade/campaign/cesm/development/lmwg/landuse_source_data/CTSM53RawData/globalctsm53histMKSRFDeg025_240709/mksrf_landuse_ctsm53_soilcolor_CLIM.c240709.nc</data_filename>
```

Comment:
Same thing as above.

Reply:
All files under /CTSM53RawData are covered by my previous comment, so I will resolve this one.